### PR TITLE
fixed typos and grammatical errors

### LINKED ISSUE #567: typos and grammatical errors here and there
## Description

I found some grammatical errors and typos 
## Resolution

I have resolved many of these.

### PR DESCRIPTION
**Pull request by palak199**
## Fixes

Fixes #567 that I opened

### CODE REVIEW

File: .storybook/meta/overview.stories.mdx [hunk 1]
```diff
@@ -18,7 +18,7 @@ The overall Vocabulary project consists of
 ## Why does Vocabulary exist?
 
 We aim to have a consistency in design and development (that works as the base of CC brand) across all major Creative
-create a cohesive brand identity for CC, which will both improve user experience and drive more awareness of our work
+create a cohesive brand identity for CC, which will improve both user experience and drive more awareness of our work
```
Comment (dhruvkb) [inline .storybook/meta/overview.stories.mdx]: I don't agree, the word 'both' here applies to "improve user experience" and "drive more awareness" so it should precede both of those phrases.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Looks good except one change!

File: .storybook/meta/overview.stories.mdx (hunk 1 above)
Comment (palak199) [reply .storybook/meta/overview.stories.mdx]: @dhruvkb I have placed it preceding two both of the phases. Rest if you still find it inappropriate, my apologies, i ll change it and submit again.

Comment (palak199) [reply .storybook/meta/overview.stories.mdx]: @dhruvkb so should I modify it to what it was before ?

Comment (dhruvkb) [reply .storybook/meta/overview.stories.mdx]: Yeah, I still feel that the old one was more accurate. Please switch it back.

**Review by Dhruvi16 — APPROVED** [review]

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**